### PR TITLE
Dance page - style Amazon logo section

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is to style the amazon log section

Implementation
- [x] Make section responsive down to 320px
- [x] Remove colon from In partnership with
- [x] Resized logo to 250 px
- [x] Add space between In partnership with and logo
- [x] line up text with logo

### screen shot
screen shot 2018-11-09 at 9 49 50 am

- Before
screen shot 2018-11-09 at 4 09 40 am

- Before (at smaller width)
screen shot 2018-11-09 at 4 09 50 am

- After
screen shot 2018-11-09 at 3 53 24 am

-After (at smaller width)

screen shot 2018-11-09 at 9 55 44 am

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css [hunk 1]
```diff
@@ -15,3 +15,13 @@
...
+
+@media screen and (min-width: 0px) and (max-width: 512px) {
+  .sponsor-logo-with-text {
+    flex-direction: column;
+  }
+
+  .sponsor-logo-with-text .sponsor-logo-image {
+    margin-top: 10px;
+  }
+}
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css]: Nit: Missing new line at the end of the file

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 %link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/dance-landing/dance-sponsor-logo.css'}
 
-%div.sponsor-logo-with-text
-  %p= I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
-  %div
-    %img{src:"/images/fit-450/AFE-Dark-Orange-logo.png"}
+%div.sponsor-logo-with-text{style: "font-size: 24px; text-align: center; margin-top: 60px"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml]: Nit: I like to keep my HAML as clean as possible. If there is an id or class, you can remove the %div part.

Also, these in-line styles can be moved to your css file.

**Comment by tanyaparker** [comment]
Your checklist in your PR description is unchecked. Are you still working on things or is it ready for review? For example, you said you removed the colon but the After screenshot still has it.

**Comment by nkiruka** [comment]
@tanyaparker It will be visible after content scoop today at noon.  I will update PR with screen shot.  The i18n gsheet has been updated.

**Comment by tanyaparker** [comment]
You could either 1) ssh into staging, scoop it, merge staging into your branch again or 2) temporarily edit your pegasus/cache/i18n/en-US.yml file with the same change but don't commit it so that you can preview your changes locally.

So are you waiting to check off the checklist until after you merge your PR?

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml (hunk 2 above)
Comment (Erin007) [reply pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml]: Same for line 6 and throughout the file.

**Comment by nkiruka** [comment]
@tanyaparker updated i18n strings have been merged onto staging, updated code per feedback.  Ready for review.  Thanks

File: pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css [hunk 3]
```diff
@@ -2,15 +2,27 @@
...
   text-align: center;
   padding-top: 20px;
-  margin-left: 5%;
-  margin-right: 5%;
+  margin-bottom: 85px;
+  margin-left: 18%;
+  margin-right: 18%;
+}
+
+@media screen and (min-width: 0px) and (max-width: 512px) {
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css]: I don't think you need to have a separate css file just for the logo classes. I'd put this inside the dance.css file which should have all the styles needed for /dance.

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css]: done!

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml [hunk 4]
```diff
@@ -1,9 +1,10 @@
-%link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/dance-landing/dance-sponsor-logo.css'}
+%link{href: "/css/dance-landing/dance.css", type: "text/css", rel: "stylesheet"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml:1]: You don't need to load this again since it's already loaded on dance.haml.

**Pull request merged**